examples/api-routes-apollo-server-and-client-auth/lib/user.js

```diff
@@ -37,11 +37,10 @@ export async function findUser({ email }) {
 
 // Compare the password of an already fetched user (using `findUser`) and compare the
 // password for a potential match
-export async function validatePassword(user, inputPassword) {
-  const password = crypto
-    .pbkdf2Sync(user.hash, user.salt, 1000, 64, 'sha512')
+export const validatePassword = (user, inputPassword) => {
```

Comment:
It's generally better to use function instead of a const assigned to an anonymous function here

Reply:
Switched to a function